src/resources/postcss/tickets-forms.pcss

```diff
@@ -3,15 +3,11 @@
  *
  * @since 5.0.0
  */
-
- @import "utilities/_all.pcss";
-
```

Comment:
if we remove this where do we get the variables from?

Reply:
from common-skeleton and common-full